drgn/helpers/linux/runq.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2025, Oracle and/or its affiliates.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+"""
+runq
+-------------
+
+The ``drgn.helpers.linux.runq`` module provides helpers for working with the
+Linux runq.
```

Comment:
A few nits and suggestions on the docstring, since this does appear in the public docs.

- Need to match the underline length to the title above it. I think it triggers a Sphinx warning if not.
- In both the title and the sentence below, I'd suggest not using "runq". That's just a crash command name, not the name of a subsystem or structure. Instead, "run queues".

```suggestion
Run Queues
----------

The ``drgn.helpers.linux.runq`` module provides helpers for working with the
Linux run queues.
```